lib/devices/target.ml

```diff
@@ -215,7 +215,7 @@ module Solo5 = struct
 
   let build_packages =
     [
-      Functoria.package ~min:"0.8.2" ~max:"2.0.0" ~scope:`Switch ~build:true
+      Functoria.package ~min:"0.8.2" ~max:"1.0.0" ~scope:`Switch ~build:true
```

Comment:
Oops, a late request, would it be possible to reactivate ocaml-solo5 1+ once the new release of opam-monorepo is available?
```suggestion
      Functoria.package ~min:"0.8.2" ~max:"2.0.0" ~scope:`Switch ~build:true
```

Reply:
sure. good that you noticed that.